docs/make-partition.md

```diff
@@ -0,0 +1,194 @@
+-A INPUT -p tcp -s 127.0.0.1 --sport 3000 -d 127.0.0.1 --dport 36113 -j REJECT --reject-with tcp-reset
+COMMIT
+```
+
+To sum up:
+
+* New connections to the listening ports (`3000`, `3001`) will be blocked. This
+  prevents tchannel to re-open new valid connections.
+* Relevant existing connections will be terminated (e.g. `3000` to ephemeral
+  ports).
+* Linux only: for the above to work, the firewall needs to explicitly accept
+  `RST` packets.
+
+During the partition, new connections to the nodes will be impossible to make.
+This is important to keep in mind when using `ringpop-admin`: **invoke
+ringpop-admin before forming the partition**.
+
+Armed with background how this works, we can go and make a local partition:
+
+## Start the tick-cluster
+
+In this example, we use Node version of ringpop, but we can use `testpop` from
+go too:
+
+```shell
+$ ./scripts/tick-cluster.js -n 4 ./main.js  # node
+```
+
+## Open the ringpop-admin to observe the cluster state
+
+We shall open `ringpop-admin partitions` and `ringpop-admin top` before making the
+partition. This way, the "management" connections will be open and status will
+be visible during the partition:
+
+```shell
+$ ringpop-admin top 127.0.0.1:3000
+$ ringpop-admin partitions -w 1 127.0.0.1:3000  # other terminal
+```
+
+`ringpop-admin top` will show something like this:
+
+```
+Address          P1
+127.0.0.1:3000   alive
+127.0.0.1:3001   alive
+127.0.0.1:3002   alive
+127.0.0.1:3003   alive
+1 of 4
+```
+
+`ringpop-admin partitions` will show a single partition, updated every second:
+
+```
+10:27:09.615   Checksum    # Nodes   # Alive   # Suspect   # Faulty   Sample Host
+               192859590   4         4         0           0          127.0.0.1:3000
+10:27:10.607   Checksum    # Nodes   # Alive   # Suspect   # Faulty   Sample Host
+               192859590   4         4         0           0          127.0.0.1:3000
+```
+
+## Start the partition
+
+First, check how the firewall rules would look like before applying them to the
+firewall (optionally, you can pass `--platform=darwin` or `--platform=linux` to
+the `make_partition` script:
+
+```shell
+$ sudo lsof -Pnni | ./tools/make_partition 3000,3001 3002,3003
+```
+
+If you are happy with the output, apply the rules:
+
+OS X:
+
+```shell
+$ sudo lsof -Pnni | ./tools/make_partition 3000,3001 3002,3003 | sudo pfctl -emf -
+```
+
+Linux:
+
+```
+$ sudo lsof -Pnni | ./tools/make_partition 3000,3001 3002,3003 | sudo iptables-restore
+```
+
+In a few seconds, you should see output from `tick-cluster` that some of the
+nodes aren't able to ping each other. Let's verify we actuall have a partition.
+
+## Checking in the tools
+
+On partition, `ringpop-admin top` (opened before the partition) should display
+something like this:
+
+```
+Address          P1       P2
+127.0.0.1:3000   faulty   alive
+127.0.0.1:3001   faulty   alive
+127.0.0.1:3002   alive    faulty
+127.0.0.1:3003   alive    faulty
```

Comment:
This will not work with a short reaping-faulty-node timeout! :'(

I think it's a problem.. Do you think there is an easy way to extend the faulty node timeout? I guess we still want to be able to simulate the partitions and healing..